docs/tutorial/cpp/src/main_game_logic.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright © SixtyFPS GmbH <info@slint-ui.com>
-// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.0 OR LicenseRef-Slint-commercial
```

Comment:
The tutorial should probably be MIT.
And maybe the other docs too.

Reply:
like CC-BY-4.0 https://spdx.org/licenses/CC-BY-4.0.html